File: app/src/main/java/com/another1dd/cybersportnews/activities/MainActivity.java
package com.another1dd.cybersportnews.activities;

import android.content.SharedPreferences;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentStatePagerAdapter;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.Menu;
import android.view.MenuItem;

import com.another1dd.cybersportnews.fragments.RecyclerViewFragment;
import com.github.florent37.materialviewpager.MaterialViewPager;

import java.util.ArrayList;
import java.util.LinkedHashMap;
import java.util.Map;

public class MainActivity extends AppCompatActivity {

    MaterialViewPager materialViewPager;
    LinkedHashMap<String[], Boolean> tabs = new LinkedHashMap<>();
    final static String DOTA_IC = "Dota 2";
    final static String CSGO_IC = "Counter-Strike GO";
    final static String HEARTHSTONE_IC = "Hearthstone";
    final static String LOL_IC = "League Of Legends";
    final static String WOT_IC = "World Of Tanks";
    final static String HOTS_IC = "Heroes Of The Storm";
    final static String STARCRAFT_IC = "Starcraft 2";
    final static String OVERWATCH_IC = "Overwatch";
    final static String OTHER_IC = "Другое";
    final static String LIFE_IC = "Life";
    final static String[] DOTA = new String[]{"http://www.cybersport.ru/news/?game=21&MUL_MODE=", DOTA_IC};
    final static String[] CSGO = new String[]{"http://www.cybersport.ru/news/?game=19&MUL_MODE=", CSGO_IC};
    final static String[] HEARTHSTONE = new String[]{"http://www.cybersport.ru/news/?game=86659&MUL_MODE=", HEARTHSTONE_IC};
    final static String[] LOL = new String[]{"http://www.cybersport.ru/news/?game=23955&MUL_MODE=", LOL_IC};
    final static String[] WOT = new String[]{"http://www.cybersport.ru/news/?game=22&MUL_MODE=", WOT_IC};
    final static String[] HOTS = new String[]{"http://www.cybersport.ru/news/?game=108725&MUL_MODE=", HOTS_IC};
    final static String[] STARCRAFT = new String[]{"http://www.cybersport.ru/news/?game=23677&MUL_MODE=", STARCRAFT_IC};
    final static String[] OVERWATCH = new String[]{"http://www.cybersport.ru/news/?game=134502&MUL_MODE=", OVERWATCH_IC};
    final static String[] OTHER = new String[]{"http://www.cybersport.ru/news/?game=41377&MUL_MODE=", OTHER_IC};
    final static String[] LIFE = new String[]{"http://www.cybersport.ru/news/?game=102405&MUL_MODE=", LIFE_IC};
    final static String APP_PREF = "spref_tabs";
    int tabCount = 0;
    SharedPreferences sharedPreferences;


    final static String MAIN_HTTP = "http://www.cybersport.ru/news/";


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(com.another1dd.cybersportnews.R.layout.activity_main);

        sharedPreferences = getSharedPreferences(APP_PREF, MODE_PRIVATE);


        tabs.put(DOTA, sharedPreferences.getBoolean(DOTA_IC, true));
            tabs.put(CSGO, sharedPreferences.getBoolean(CSGO_IC, true));
            tabs.put(HEARTHSTONE, sharedPreferences.getBoolean(HEARTHSTONE_IC, true));
            tabs.put(LOL, sharedPreferences.getBoolean(LOL_IC, true));
            tabs.put(WOT, sharedPreferences.getBoolean(WOT_IC, false));
            tabs.put(HOTS, sharedPreferences.getBoolean(HOTS_IC, false));
            tabs.put(STARCRAFT, sharedPreferences.getBoolean(STARCRAFT_IC, false));
            tabs.put(OVERWATCH, sharedPreferences.getBoolean(OVERWATCH_IC, false));
            tabs.put(OTHER, sharedPreferences.getBoolean(OTHER_IC, false));
            tabs.put(LIFE, sharedPreferences.getBoolean(LIFE_IC, false));

        tabCount = tabs.size();

        materialViewPager = findViewById(com.another1dd.cybersportnews.R.id.materialViewPager);

        Toolbar toolbar = materialViewPager.getToolbar();

        if (toolbar != null) {
            setSupportActionBar(toolbar);

            ActionBar actionBar = getSupportActionBar();
            actionBar.setDisplayHomeAsUpEnabled(false);
            actionBar.setDisplayShowHomeEnabled(false);
            actionBar.setDisplayShowTitleEnabled(true);
            actionBar.setDisplayUseLogoEnabled(false);
            actionBar.setHomeButtonEnabled(false);
        }

        updateMaterialViewPager();
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(com.another1dd.cybersportnews.R.menu.menu_main, menu);
        MenuItem dota_menu = menu.findItem(com.another1dd.cybersportnews.R.id.dota_menu);
        MenuItem csgo_menu = menu.findItem(com.another1dd.cybersportnews.R.id.csgo_menu);
        MenuItem heathstone_menu = menu.findItem(com.another1dd.cybersportnews.R.id.hearthstone_menu);
        MenuItem lol_menu = menu.findItem(com.another1dd.cybersportnews.R.id.lol_menu);
        MenuItem wot_menu = menu.findItem(com.another1dd.cybersportnews.R.id.wot_menu);
        MenuItem hots_menu = menu.findItem(com.another1dd.cybersportnews.R.id.hots_menu);
        MenuItem sc2_menu = menu.findItem(com.another1dd.cybersportnews.R.id.sc2_menu);
        MenuItem overwatch_menu = menu.findItem(com.another1dd.cybersportnews.R.id.overwatch_menu);
        MenuItem other_menu = menu.findItem(com.another1dd.cybersportnews.R.id.other_menu);
        MenuItem life_menu = menu.findItem(com.another1dd.cybersportnews.R.id.life_menu);

        if (tabs.get(DOTA)) {
            dota_menu.setChecked(true);
        } else {
            dota_menu.setChecked(false);
        }

        if (tabs.get(CSGO)) {
            csgo_menu.setChecked(true);
        } else {
            csgo_menu.setChecked(false);
        }

        if (tabs.get(HEARTHSTONE)) {
            heathstone_menu.setChecked(true);
        } else {
            heathstone_menu.setChecked(false);
        }

        if (tabs.get(LOL)) {
            lol_menu.setChecked(true);
        } else {
            lol_menu.setChecked(false);
        }

        if (tabs.get(WOT)) {
            wot_menu.setChecked(true);
        } else {
            wot_menu.setChecked(false);
        }

        if (tabs.get(HOTS)) {
            hots_menu.setChecked(true);
        } else {
            hots_menu.setChecked(false);
        }

        if (tabs.get(STARCRAFT)) {
            sc2_menu.setChecked(true);
        } else {
            sc2_menu.setChecked(false);
        }

        if (tabs.get(OVERWATCH)) {
            overwatch_menu.setChecked(true);
        } else {
            overwatch_menu.setChecked(false);
        }

        if (tabs.get(OTHER)) {
            other_menu.setChecked(true);
        } else {
            other_menu.setChecked(false);
        }

        if (tabs.get(LIFE)) {
            life_menu.setChecked(true);
        } else {
            life_menu.setChecked(false);
        }
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {

        switch (item.getItemId()) {
            case com.another1dd.cybersportnews.R.id.dota_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(DOTA, false);
                    SavePreferences(DOTA_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(DOTA, true);
                    SavePreferences(DOTA_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.csgo_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(CSGO, false);
                    SavePreferences(CSGO_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(CSGO, true);
                    SavePreferences(CSGO_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.hearthstone_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(HEARTHSTONE, false);
                    SavePreferences(HEARTHSTONE_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(HEARTHSTONE, true);
                    SavePreferences(HEARTHSTONE_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.lol_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(LOL, false);
                    SavePreferences(LOL_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(LOL, true);
                    SavePreferences(LOL_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.wot_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(WOT, false);
                    SavePreferences(WOT_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(WOT, true);
                    SavePreferences(WOT_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.hots_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(HOTS, false);
                    SavePreferences(HOTS_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(HOTS, true);
                    SavePreferences(HOTS_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.sc2_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(STARCRAFT, false);
                    SavePreferences(STARCRAFT_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(STARCRAFT, true);
                    SavePreferences(STARCRAFT_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.overwatch_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(OVERWATCH, false);
                    SavePreferences(OVERWATCH_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(OVERWATCH, true);
                    SavePreferences(OVERWATCH_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.other_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(OTHER, false);
                    SavePreferences(OTHER_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(OTHER, true);
                    SavePreferences(OTHER_IC, true);
                }
                updateMaterialViewPager();
                return true;
            case com.another1dd.cybersportnews.R.id.life_menu:
                if (item.isChecked()) {
                    item.setChecked(false);
                    tabs.put(LIFE, false);
                    SavePreferences(LIFE_IC, false);
                } else {
                    item.setChecked(true);
                    tabs.put(LIFE, true);
                    SavePreferences(LIFE_IC, true);
                }
                updateMaterialViewPager();
                return true;
            default:
                return super.onOptionsItemSelected(item);
        }

    }




    private void updateMaterialViewPager() {
        final ArrayList<String[]> arrayList = new ArrayList<>();

        for (Map.Entry<String[], Boolean> map : tabs.entrySet()) {
            if (map.getValue()) {
                arrayList.add(map.getKey());
            }
        }


        tabCount = arrayList.size();


        materialViewPager.getViewPager().setAdapter(new FragmentStatePagerAdapter(getSupportFragmentManager()) {

            @Override
            public Fragment getItem(int position) {
                switch (position) {
                    case 0:
                        return RecyclerViewFragment.newInstance(arrayList.get(0)[0]);
                    case 1:
                        return RecyclerViewFragment.newInstance(arrayList.get(1)[0]);
                    case 2:
                        return RecyclerViewFragment.newInstance(arrayList.get(2)[0]);
                    case 3:
                        return RecyclerViewFragment.newInstance(arrayList.get(3)[0]);
                    case 4:
                        return RecyclerViewFragment.newInstance(arrayList.get(4)[0]);
                    case 5:
                        return RecyclerViewFragment.newInstance(arrayList.get(5)[0]);
                    case 6:
                        return RecyclerViewFragment.newInstance(arrayList.get(6)[0]);
                    case 7:
                        return RecyclerViewFragment.newInstance(arrayList.get(7)[0]);
                    case 8:
                        return RecyclerViewFragment.newInstance(arrayList.get(8)[0]);
                    case 9:
                        return RecyclerViewFragment.newInstance(arrayList.get(9)[0]);
                    default:
                        return RecyclerViewFragment.newInstance(MAIN_HTTP);
                }


            }

            @Override
            public int getCount() {
                return tabCount;
            }


            @Override
            public CharSequence getPageTitle(int position) {
                switch (position) {
                    case 0:
                        return arrayList.get(0)[1];
                    case 1:
                        return arrayList.get(1)[1];
                    case 2:
                        return arrayList.get(2)[1];
                    case 3:
                        return arrayList.get(3)[1];
                    case 4:
                        return arrayList.get(4)[1];
                    case 5:
                        return arrayList.get(5)[1];
                    case 6:
                        return arrayList.get(6)[1];
                    case 7:
                        return arrayList.get(7)[1];
                    case 8:
                        return arrayList.get(8)[1];
                    case 9:
                        return arrayList.get(9)[1];
                    default:
                        return "Page " + position;
                }
            }
        });


        materialViewPager.getViewPager().setOffscreenPageLimit(tabCount);

        materialViewPager.getPagerTitleStrip().setViewPager(materialViewPager.getViewPager());
    }

    private void SavePreferences(String key, Boolean value) {
        sharedPreferences = getSharedPreferences(APP_PREF, MODE_PRIVATE);
        SharedPreferences.Editor editor = sharedPreferences.edit();
        editor.putBoolean(key, value);
        editor.apply();
    }



}

File: app/src/main/java/com/another1dd/cybersportnews/activities/MainActivity.kt
package com.another1dd.cybersportnews.activities

import android.content.Context
import android.content.SharedPreferences
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.another1dd.cybersportnews.R
import com.another1dd.cybersportnews.constants.CybersportNews
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity() {
    private var tabs = LinkedHashMap<Array<String>, Boolean>()
    internal var tabCount = 0
    private lateinit var sharedPreferences: SharedPreferences


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        sharedPreferences = getSharedPreferences(CybersportNews.APP_PREF, Context.MODE_PRIVATE)


        tabs.put(CybersportNews.DOTA, sharedPreferences.getBoolean(CybersportNews.DOTA_IC, true))
        tabs.put(CybersportNews.CSGO, sharedPreferences.getBoolean(CybersportNews.CSGO_IC, true))
        tabs.put(CybersportNews.HEARTHSTONE, sharedPreferences.getBoolean(CybersportNews.HEARTHSTONE_IC, true))
        tabs.put(CybersportNews.LOL, sharedPreferences.getBoolean(CybersportNews.LOL_IC, true))
        tabs.put(CybersportNews.WOT, sharedPreferences.getBoolean(CybersportNews.WOT_IC, false))
        tabs.put(CybersportNews.HOTS, sharedPreferences.getBoolean(CybersportNews.HOTS_IC, false))
        tabs.put(CybersportNews.STARCRAFT, sharedPreferences.getBoolean(CybersportNews.STARCRAFT_IC, false))
        tabs.put(CybersportNews.OVERWATCH, sharedPreferences.getBoolean(CybersportNews.OVERWATCH_IC, false))
        tabs.put(CybersportNews.OTHER, sharedPreferences.getBoolean(CybersportNews.OTHER_IC, false))
        tabs.put(CybersportNews.LIFE, sharedPreferences.getBoolean(CybersportNews.LIFE_IC, false))

        tabCount = tabs.size


        setSupportActionBar(mainToolbar)

        val actionBar = supportActionBar
        actionBar?.setDisplayHomeAsUpEnabled(false)
        actionBar?.setDisplayShowHomeEnabled(false)
        actionBar?.setDisplayShowTitleEnabled(true)
        actionBar?.setDisplayUseLogoEnabled(false)
        actionBar?.setHomeButtonEnabled(false)


//        updateMaterialViewPager()
    }

    /*override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(com.another1dd.cybersportnews.R.menu.menu_main, menu)
        val dotaMenu = menu.findItem(com.another1dd.cybersportnews.R.id.dotaMenu)
        val csgoMenu = menu.findItem(com.another1dd.cybersportnews.R.id.csGoMenu)
        val heathstoneMenu = menu.findItem(com.another1dd.cybersportnews.R.id.hearthstoneMenu)
        val lolMenu = menu.findItem(com.another1dd.cybersportnews.R.id.lolMenu)
        val wotMenu = menu.findItem(com.another1dd.cybersportnews.R.id.wotMenu)
        val hotsMenu = menu.findItem(com.another1dd.cybersportnews.R.id.hotsMenu)
        val sc2Menu = menu.findItem(com.another1dd.cybersportnews.R.id.sc2Menu)
        val overwatchMenu = menu.findItem(com.another1dd.cybersportnews.R.id.overwatchMenu)
        val otherMenu = menu.findItem(com.another1dd.cybersportnews.R.id.otherMenu)
        val lifeMenu = menu.findItem(com.another1dd.cybersportnews.R.id.lifeMenu)

        dotaMenu.isChecked = tabs[CybersportNews.DOTA]!!
        csgoMenu.isChecked = tabs[CybersportNews.CSGO]!!
        heathstoneMenu.isChecked = tabs[CybersportNews.HEARTHSTONE]!!
        lolMenu.isChecked = tabs[CybersportNews.LOL]!!
        wotMenu.isChecked = tabs[CybersportNews.WOT]!!
        hotsMenu.isChecked = tabs[CybersportNews.HOTS]!!
        sc2Menu.isChecked = tabs[CybersportNews.STARCRAFT]!!
        overwatchMenu.isChecked = tabs[CybersportNews.OVERWATCH]!!
        otherMenu.isChecked = tabs[CybersportNews.OTHER]!!
        lifeMenu.isChecked = tabs[CybersportNews.LIFE]!!
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

        when (item.itemId) {
            R.id.dotaMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.DOTA, false)
                    SavePreferences(CybersportNews.DOTA_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.DOTA, true)
                    SavePreferences(CybersportNews.DOTA_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.csGoMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.CSGO, false)
                    SavePreferences(CybersportNews.CSGO_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.CSGO, true)
                    SavePreferences(CybersportNews.CSGO_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.hearthstoneMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.HEARTHSTONE, false)
                    SavePreferences(CybersportNews.HEARTHSTONE_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.HEARTHSTONE, true)
                    SavePreferences(CybersportNews.HEARTHSTONE_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.lolMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.LOL, false)
                    SavePreferences(CybersportNews.LOL_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.LOL, true)
                    SavePreferences(CybersportNews.LOL_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.wotMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.WOT, false)
                    SavePreferences(CybersportNews.WOT_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.WOT, true)
                    SavePreferences(CybersportNews.WOT_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.hotsMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.HOTS, false)
                    SavePreferences(CybersportNews.HOTS_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.HOTS, true)
                    SavePreferences(CybersportNews.HOTS_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.sc2Menu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.STARCRAFT, false)
                    SavePreferences(CybersportNews.STARCRAFT_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.STARCRAFT, true)
                    SavePreferences(CybersportNews.STARCRAFT_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            R.id.overwatchMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.OVERWATCH, false)
                    SavePreferences(CybersportNews.OVERWATCH_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.OVERWATCH, true)
                    SavePreferences(CybersportNews.OVERWATCH_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            com.another1dd.cybersportnews.R.id.otherMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.OTHER, false)
                    SavePreferences(CybersportNews.OTHER_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.OTHER, true)
                    SavePreferences(CybersportNews.OTHER_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            com.another1dd.cybersportnews.R.id.lifeMenu -> {
                if (item.isChecked) {
                    item.isChecked = false
                    tabs.put(CybersportNews.LIFE, false)
                    SavePreferences(CybersportNews.LIFE_IC, false)
                } else {
                    item.isChecked = true
                    tabs.put(CybersportNews.LIFE, true)
                    SavePreferences(CybersportNews.LIFE_IC, true)
                }
                updateMaterialViewPager()
                return true
            }
            else -> return super.onOptionsItemSelected(item)
        }

    }


    private fun updateMaterialViewPager() {
        val arrayList = ArrayList<Array<String>>()

        for ((key, value) in tabs) {
            if (value) {
                arrayList.add(key)
            }
        }


        tabCount = arrayList.size


        materialViewPager.viewPager.adapter = object : FragmentStatePagerAdapter(supportFragmentManager) {

            override fun getItem(position: Int): Fragment {
                return when (position) {
                    0 -> RecyclerViewFragment.newInstance(arrayList[0][0])
                    1 -> RecyclerViewFragment.newInstance(arrayList[1][0])
                    2 -> RecyclerViewFragment.newInstance(arrayList[2][0])
                    3 -> RecyclerViewFragment.newInstance(arrayList[3][0])
                    4 -> RecyclerViewFragment.newInstance(arrayList[4][0])
                    5 -> RecyclerViewFragment.newInstance(arrayList[5][0])
                    6 -> RecyclerViewFragment.newInstance(arrayList[6][0])
                    7 -> RecyclerViewFragment.newInstance(arrayList[7][0])
                    8 -> RecyclerViewFragment.newInstance(arrayList[8][0])
                    9 -> RecyclerViewFragment.newInstance(arrayList[9][0])
                    else -> RecyclerViewFragment.newInstance(CybersportNews.MAIN_HTTP)
                }
            }

            override fun getCount(): Int {
                return tabCount
            }


            override fun getPageTitle(position: Int): CharSequence {
                return when (position) {
                    0 -> arrayList[0][1]
                    1 -> arrayList[1][1]
                    2 -> arrayList[2][1]
                    3 -> arrayList[3][1]
                    4 -> arrayList[4][1]
                    5 -> arrayList[5][1]
                    6 -> arrayList[6][1]
                    7 -> arrayList[7][1]
                    8 -> arrayList[8][1]
                    9 -> arrayList[9][1]
                    else -> "Page " + position
                }
            }
        }


        materialViewPager.viewPager.offscreenPageLimit = tabCount

        materialViewPager.pagerTitleStrip.setViewPager(materialViewPager.viewPager)
    }*/

    private fun SavePreferences(key: String, value: Boolean?) {
        sharedPreferences = getSharedPreferences(CybersportNews.APP_PREF, Context.MODE_PRIVATE)
        val editor = sharedPreferences.edit()
        editor.putBoolean(key, value!!)
        editor.apply()
    }
}
